lightning_pose/apps/labeled_frame_diagnostics.py

```diff
@@ -0,0 +1,249 @@
+"""Analyze predictions on labeled frames.
+
+Users select an arbitrary number of csvs (one per model) from their file system
```

Comment:
i'd delete all of these notes (from here up until the imports) and point people to the markdown docs, that way you don't need to maintain both